zerver/lib/hotspots.py

```diff
@@ -95,7 +105,7 @@ def get_next_hotspots(user: UserProfile) -> List[Dict[str, object]]:
                     "description": str(INTRO_HOTSPOTS[hotspot]["description"]),
                     "delay": 0.5,
                 }
-            ]
+            ] + ([] if "intro_draft" in seen_hotspots else intro_draft_hotspot)
```

Comment:
Doesn't the above loop already contain the `intro_draft` hotspot? It went through all the unseen hotspots and sent each of them along with its name, title, description, etc.

Reply:
Actually, hotspots are sent one after another.
> Doesn't the above loop already contain the `intro_draft` hotspot? 

Yes, it does. But we want the `intro_draft` hotspot to activate whenever the first draft is created. So, it can be possible that is the first draft is created before the previous hotspots to `intro_draft`.

Also changing the `intro_draft` position to the top of the list is not a good idea, because then it will block other hotspots until it is seen.

Reply:
@Riken-Shah I meant that there would be duplicate "intro_draft" objects on the client-side `page_params.hotspots` list if we did it this way, which is definitely not the right way to go.
![image](https://user-images.githubusercontent.com/30312566/111094548-85520900-8561-11eb-834d-46eacc53e270.png)

Reply:
Oh Right!
Then I think so we have two option:
1. Create another specific event for `intro_hotspot`.
2. Avoid appending `intro_hotspot` to `page_params.hotspots` if it already exists.

WDYT @sumanthvrao we should go with?

Reply:
I have implemented the second option.
Can you review @sumanthvrao?